warehouse/router/upload_stats.go

```diff
@@ -110,34 +109,27 @@ func (job *UploadJob) generateUploadAbortedMetrics() {
 
 func (job *UploadJob) recordTableLoad(tableName string, numEvents int64) {
 	rudderAPISupportedEventTypes := []string{"tracks", "identifies", "pages", "screens", "aliases", "groups"}
-	if slices.Contains(rudderAPISupportedEventTypes, strings.ToLower(tableName)) {
-		// record total events synced (ignoring additional row synced to the event table for e.g.track call)
-		job.counterStat(`event_delivery`, warehouseutils.Tag{
-			Name:  "tableName",
-			Value: strings.ToLower(tableName),
-		}).Count(int(numEvents))
+	if !slices.Contains(rudderAPISupportedEventTypes, strings.ToLower(tableName)) {
```

Comment:
```suggestion
    capturedTableName := strings.ToLower(tableName)
	if !slices.Contains(rudderAPISupportedEventTypes, capturedTableName) {
```

and use `capturedTableName` on the code bellow. To avoid any confusion and prevent tableName being used/passed by mistake in elsewhere.